benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -208,6 +216,7 @@ class GraphicsBenchmarkApp
     std::shared_ptr<KnobCheckbox>              pFullscreenQuadsSingleRenderpass;
     std::shared_ptr<KnobCheckbox>              pAlphaBlend;
     std::shared_ptr<KnobCheckbox>              pDepthTestWrite;
+    std::shared_ptr<KnobCheckbox>              pAllTexturesToWhite;
```

Comment:
probably better to rename to pAllTexuresTo1x1

Reply:
OK, done